doc/source/tutorials/animations.rst

```diff
@@ -0,0 +1,167 @@
+.. _animations:
+
+.. index:: animations
+
+Creating an animation using Emperor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this tutorial we describe how to create a principal coordinates analysis
+(PCoA) plot, and display animated traces of the samples sorted by a metadata
+category. To do this, we will describe a `Synthetic Example` (explaining
```

Comment:
To do this -> Here | For this purpose,  ... Just cause that's how the next paragraph starts.